modules/dnsmasq_dhcp/dhcp.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	"github.com/netdata/go-orchestrator/module"
 
-	"github.com/netdata/go.d.plugin/modules/dnsmasq_dhcp/ip"
+	"github.com/netdata/go.d.plugin/pkg/ip"
```

Comment:
I try to follow the order:

```
std lib

local packages (within-repo)

remote packages
```

Reply:
`github.com/netdata/go-orchestrator/module` is remote atm, `github.com/netdata/go.d.plugin/pkg/ip` - local.

Reply:
Thank you for the tip, I fixed the order inside this file and I will move forward with this pattern.